docs/monitoring/inventory.md

```diff
@@ -1,13 +1,107 @@
+- Number of different OS distributions,
+- Number of different OS versions,
+- A number of different kernel versions.
+
+![Inventory Info](../images/monitoring/inventory_inventory_info.png)
+
+### Inventory Overview
+
+Inventory overview provides per host view of each of the hosts, virtual machines and containers along with the operating system and its version, number of CPUs and their type, memory, number of hard disk drives and their total capacity. 
+
+![Inventory Overview](../images/monitoring/inventory_overview.png)
+
+Expanding each row gives detailed information about the system including:
+
+- Unique identifier of the system,
+- OS version,
+- Kernel version,
+- Java virtual machine version,
+- System type (bare metal, virtual machine, container),
+- Total memory size, 
+- Memory slot information,
+- Total disk size, 
+- Hard disk drives information.
+
+In addition view of the packages is available giving a view over the packages installed on a given system including:
+
+- Installation source, 
+- Package name, 
+- Package version.
+
+![Inventory Overview](../images/monitoring/inventory_packages.png)
+
+### Comparing Hosts
+
+The Inventory UI allows you to compare different systems to see the differences between them quickly and efficiently:
+
+![Inventory Package Comparison](../images/monitoring/inventory_package_comparison.png)
+
+## How does it Work?
+
+Inventory Monitoring part of the Monitoring Agent keeps track of the changes done on the file system when installing, upgrading or removing packages on the host system, in the virtual machine or in the container when using coming from:
+
+- Node.JS (when using NPM package manager)
+- Python (when using PIP package manager)
+- DEB (for example, when using APT package manager)
+
+In addition, the Monitoring Agent periodically checks the state of the packages and ships that information to Sematext Cloud to ensure that the state of packages is always up to date and visible in the UI.
+
+## Enabling Inventory Monitoring
+
+To enable inventory monitoring you need to adjust your `st-agent.yml` file located in the `/opt/spm/properties` directory and set the following section:
+
+```
+pkg:
+  enabled: true
+  interval: 10m
+```
+
+After that the Monitoring Agent needs to be restarted by running the following command:
+
+```
+service spm-monitor-st-agent restart
+```
+
+## Gathered Data
+
+Sematext Monitoring Agent gathers the following data about the system:
+
+- OS version,
+- Kernel version,
+- Java virtual machine version,
+- System type (bare metal, virtual machine, container),
+- Total memory size, 
+- Memory slot information,
+- Total disk size, 
+- Hard disk drives information.
+
+In addition, the following information about the packages is gathered for NodeJS, Python and DEB sources:
+
+- Installation source, 
+- Package name, 
+- Package version.
+
+## Solving Problems With Inventory Monitoring
+
+The Inventory monitoring allows you to quickly solve the following issues:
```

Comment:
The Inventory monitoring allows you to quickly solve the following issues:
->
Inventory monitoring allows quickly solving the following issues:

Reply:
Done.